cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -295,7 +295,7 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Create Index",
       text =
-        """The `CreateIndex` operator creates an index on a property for all nodes having a certain label.
+        """The `CreateIndex` operator creates an index on a property for all nodes or relationships having a certain label or relationship type.
```

Comment:
It looks like most of the execution plans are missing in the 4.3 preview (compare https://neo4j.com/docs/cypher-manual/4.3-preview/execution-plans/operators/ and https://neo4j.com/docs/cypher-manual/current/execution-plans/operators/), so I could not really review this part. I have notified the docs team about it.